shared/styles/style-guide.desktop.js

```diff
@@ -50,6 +51,15 @@ const flexBoxCommon = {
   display: 'flex',
 }
 
+export const globalMargins = {
```

Comment:
this is fine for now but we should shoot for all the imports on top and exports on the bottom. you can define it here (since you need it) but you can export it separately below (don't need to fix this for this pr , just to discuss)